server/api/assessment/index.ts

```diff
@@ -42,8 +43,6 @@ export const AssessmentApi = {
     )
 
     // TableData
-    express.get(ApiEndPoint.Assessment.TableData.one(), AuthMiddleware.requireView, getTableData)
-
     express.get(ApiEndPoint.Assessment.TableData.one(), AuthMiddleware.requireView, getTableData)
```

Comment:
you still need to pass section to the api, otherwise the auth middleware won't work

Reply:
but requireView doesn't use section...